src/privileged/trackers/api.js

```diff
@@ -6,8 +6,6 @@ ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
 
 const { EventManager, EventEmitter } = ExtensionCommon;
 
-const {Management: {global: {tabTracker}}} = ChromeUtils.import("resource://gre/modules/Extension.jsm", {});
```

Comment:
Yeah, it's annoying that these share the same namespace, but could we consider making this `var` instead? I feel a bit uneasy not importing the thing here...